pages/research_software.md

```diff
@@ -2,8 +2,28 @@
 title: Research Software
 ---
 
-This page will list a definition of research software and how different perspectives exist
+# What is Research Software
 
-## Three tier model
+There have been various definitions of research software - the following two definitions offer two ends of a spectrum of what research software is. The RSQKit does not mandate the definition of what research software is, it taps into existing definitions - however its authors and editorial board may be opinionated on what they believe the definition is and this may vary over time and be reflected in the RSQKit pages. 
```

Comment:
```suggestion
There have been various definitions of research software - the following two definitions offer two ends of a spectrum of what research software is. The RSQKit does not mandate the definition of what research software is, however its authors and editorial board may be opinionated on what they believe the definition is and this may vary over time and be reflected in the RSQKit pages. 
```
I removed a sentence which was a little repetitivr